### ISSUE TITLE
Logitech G102 gesture button

### ISSUE THREAD

**Issue by avxkim** [mark-vandenberg/g-hub-mouse-gestures#3]
Hello, i've pasted your script just as noted in README, but couldn't make it work. I guess, wrong mappings on my gesture button on G102? Does it work in MacOS Ventura 13.0.1?

**UPDATE1**
Okay, it works, had to change:

```
gestureButtonNumber = 6;
```

It's 6 on G102. Another question, is it possible to make "pinch to zoom" just like on mx master 3 / anywhere 3 by double tapping button 6 in my case?

**Comment by mark-vandenberg** [comment]
If any of these shortcuts are sufficient for the zooming I think I could get it to work but I would need some time to develop something for the double click registration. I noticed the the shortcuts don't have the exact same functionality as pinch-to-zoom so the results in different programs may vary. If you use it in one program mainly I could make it so that the shortcut for zooming in that program will be the default here or tell you what part in the script to change to make it work exactly. 

<img width="780" alt="image" src="https://user-images.githubusercontent.com/41389220/214535639-bad29c28-aa32-4a58-9746-c772b0d5d22c.png">

**Comment by avxkim** [comment]
@mark-vandenberg it's called a Smart Zoom, if i not mistaken, you can double tap on a mac trackpad in order to smoothly zoom in, btw, it's not the same as just zooming with CMD-+, i hope you get what i mean.

On MX Anywhere 3s in Logi Options+ i had to press my middle button twice so it would fire "Smart Zoom" action.

Option + CMD + 8 seems like it is!

**Comment by mark-vandenberg** [comment]
Yeah I know what you mean with Smart Zoom. Issue is that I can't find a keyboard shortcut or another way in Lua to make that exact functionality. I think Logi Options+ can do it because it interacts with the OS on a lower level. If Option + CMD + 8 works for you I can make that but I can't promise a timeline for when I will release it.

**Comment by avxkim** [comment]
Making Opt+CMD+8 working is not easy as it seems?

**Comment by mark-vandenberg** [comment]
It's not that difficult I think, but I need to make some time for it between school, work and other hobbies 😄

**Comment by avxkim** [comment]
Waiting for you then

**Comment by avxkim** [comment]
@mark-vandenberg 

Do you know scan codes for: Left Option + Left Command? Manual doesn't have that information :(

**Comment by mark-vandenberg** [comment]
Left Option = "lalt" as in Left Alt
Left Command = "lgui" as in Left Gui so it works for both the command and windows key

**Comment by avxkim** [comment]
Thanks, can't make it work with your current script, it's kinda interferring with each other

**Comment by mark-vandenberg** [comment]
If you are still interested in this I could look into this in the coming week
